google/resource_sql_user_test.go

```diff
@@ -107,39 +125,48 @@ func testAccGoogleSqlUserDestroy(s *terraform.State) error {
 	return nil
 }
 
-func testGoogleSqlUser_basic(instance, user string) string {
+func testGoogleSqlUser_firstGen(instance, password string) string {
 	return fmt.Sprintf(`
 	resource "google_sql_database_instance" "instance" {
-		name = "i%s"
+		name = "%s"
 		region = "us-central"
 		settings {
 			tier = "D0"
 		}
 	}
 
-	resource "google_sql_user" "user" {
-		name = "user%s"
+	resource "google_sql_user" "user1" {
+		name = "john"
```

Comment:
I'm sorry to be That Person but can we use a gender-neutral name here?

Reply:
Sure. Changed for `admin`.